cache/lru_cache.cc

```diff
@@ -579,12 +576,10 @@ Cache::Handle* LRUCacheShard::Lookup(
       e->Ref();
       e->SetIsInSecondaryCache(is_in_sec_cache);
 
```

Comment:
1. For a standalone handle , `wait` can be either true or false. 
2. If `wait` is false, the Promote() in WaitAll should still promote it as a standalone handle; but 
In WaitAll(): 
shard->Promote(lru_handle, /*standalone=*/false);

Reply:
OK, oops I missed that path